modules/microshift-creating-ostree-iso.adoc

```diff
@@ -23,7 +23,7 @@ Use the following procedure to create the ISO. The {op-system-ostree} Installer
 ----
 $ BUILDID=$(sudo composer-cli compose start-ostree --ref "rhel/{op-system-version-major}/$(uname -m)/edge" __<microshift_blueprint>__ edge-container | awk '{print $2}') <1>
 ----
-<1> Replace _<microshift_blueprint>_ with the name of your blueprint.
+<1> Replace `<microshift_blueprint>` with the name of your blueprint.
```

Comment:
Probably the example should also have the italics dropped?

![Screenshot from 2024-09-02 15-59-41](https://github.com/user-attachments/assets/bf2afe56-07c6-4a68-8865-0f16110051d3)

![Screenshot from 2024-09-02 16-00-26](https://github.com/user-attachments/assets/c3df3973-3a2e-47dc-b703-b2811f4a460b)

Reply:
No, Microshift follows using italics in command. Please refer https://github.com/openshift/openshift-docs/pull/78516#discussion_r1672436701 for the discussion on the itaclics used in Microshift.